delfin/drivers/dell_emc/scaleio/rest_handler.py

```diff
@@ -0,0 +1,397 @@
+                    "type": initiators_type,
+                    "status": status,
+                    "native_storage_host_id": initiators_json.get(
+                        'protectionDomainId'),
+                }
+                initiators_list.append(initiators_dict)
+            return initiators_list
+        except exception.DelfinException as err:
+            err_msg = "Get Storage initiators error: %s" % err.msg
+            LOG.error(err_msg)
+            raise err
+        except Exception as e:
+            LOG.error("Get Storage initiators error: %s", six.text_type(e))
+            raise exception.InvalidResults(e)
+
+    def list_storage_hosts(self, storage_id):
+        host_list = []
+        try:
+            storage_hosts = self.get_rest_info(consts.REST_SCALIO_HOSTS)
+            for host_json in (storage_hosts or []):
+                status = host_json.get('mdmConnectionState')
+                if 'Connected' == status:
+                    status = constants.HostStatus.NORMAL
+                elif 'Disconnected' == status:
+                    status = constants.HostStatus.OFFLINE
+                ip_list = host_json.get('ipList')
+                ip_address = ''
+                for ip in ip_list:
+                    ip_address = ip.get('ip')
+                host_dict = {
+                    "name": host_json.get('name'),
+                    "description": host_json.get('sdcGuid'),
+                    "storage_id": storage_id,
+                    "native_storage_host_id":
+                        host_json.get('id'),
+                    "os_type": host_json.get('osType'),
+                    "status": status,
+                    "ip_address": ip_address
+                }
+                host_list.append(host_dict)
+            return host_list
+        except exception.DelfinException as err:
+            err_msg = "Get Storage hosts error: %s" % err.msg
+            LOG.error(err_msg)
+            raise err
+        except Exception as e:
+            LOG.error("Get Storage hosts error: %s", six.text_type(e))
+            raise exception.InvalidResults(e)
+
+    def list_masking_views(self, storage_id):
+        list_masking_views_list = []
+        try:
+            storage_view = self.get_rest_info(consts.REST_SCALEIO_VOLUMES)
+            for map_json in (storage_view or []):
+                view_name = map_json.get('name')
+                volume_id = map_json.get('id')
+                map_sdc_list = map_json.get('mappedSdcInfo')
+                if map_sdc_list:
+                    for map_sdc in map_sdc_list:
+                        sdc_id = map_sdc.get('sdcId')
+                        uid = uuid.uuid4()
+                        view_map = {
+                            "name": view_name + sdc_id + volume_id,
+                            "description": view_name,
+                            "storage_id": storage_id,
+                            "native_masking_view_id": str(uid) + volume_id,
```

Comment:
native_masking_view_id should be a unique value, so the data generated in this way is different.
![image](https://user-images.githubusercontent.com/59107077/172136352-1c6311d9-ad27-44f7-b466-a9a027b21f08.png)

Reply:
Replace the random ID with name plus host ID plus volume ID
![image](https://user-images.githubusercontent.com/88076879/172276142-075b3382-efd3-4155-92b7-07d1468ab9a1.png)